.jenkins/common.groovy

```diff
@@ -75,7 +75,7 @@ def runTestCommand (platform, project, boolean debug=false)
     def command = """#!/usr/bin/env bash
                 set -x
                 cd ${project.paths.project_build_prefix}/build/${directory}/clients/staging
-                ROCM_PATH=/opt/rocm GTEST_LISTENER=NO_PASS_LINE_IN_LOG ./${testBinaryName} --gtest_color=yes --R 80 --gtest_filter='-rocfft_UnitTest.repo_twiddle'
+                ROCM_PATH=/opt/rocm GTEST_LISTENER=NO_PASS_LINE_IN_LOG ./${testBinaryName} --precompile --gtest_color=yes --R 80 --gtest_filter='-rocfft_UnitTest.repo_twiddle'
```

Comment:
Not related to this PR, just a thought,  no "/opt/rocm" even in CI.

Reply:
Hmm.. yeah that shouldn't really be necessary, or at least if it is, something outside of this script should be setting it.